File: Universal Windows Platform Solution/Phantom of the West/Phantom of the West/Visual Novel/Chapters/Chapter 5/MathQuestion4IncorrectAnswer.cs
using Phantom_of_the_West.Visual_Novel.Serialization.Event_Flagging;

namespace Phantom_of_the_West.Visual_Novel.Chapters.Chapter_5
{
	internal class MathQuestion4IncorrectAnswer : StoryChoice
	{
		internal MathQuestion4IncorrectAnswer() : base(StoryChoice.GetLocalizedString("ch5MathQuestion4IncorrectAnswer"), 5050)
		{
		}

		public override void Select()
		{
			PotWVN vn = PotWVN.MainVN;
			vn.EventFlags.Ch5AnsweredMathQuestion4 = TwoChoiceAnswer.Incorrect;
			base.Select();
		}
	}
}

File: Android Project/app/src/main/java/com/christopherraleigh/phantomofthewest/visual_novel/chapters/chapter_5/MathQuestion4IncorrectAnswer.java
package com.christopherraleigh.phantomofthewest.visual_novel.chapters.chapter_5;

import com.christopherraleigh.phantomofthewest.R;
import com.christopherraleigh.phantomofthewest.visual_novel.PotWVN;
import com.christopherraleigh.phantomofthewest.visual_novel.chapters.StoryChoice;
import com.christopherraleigh.phantomofthewest.visual_novel.serialization.event_flagging.TwoChoiceAnswer;

/**
 * Created by christopherraleigh on 2016-05-18.
 */
public class MathQuestion4IncorrectAnswer extends StoryChoice {
    public MathQuestion4IncorrectAnswer() {
        super(R.string.ch5MathQuestion4IncorrectAnswer, 5050);
    }

    @Override
    public void select() {
        PotWVN vn = PotWVN.getMainVN();
        vn.getEventFlags().setCh5AnsweredMathQuestion4(TwoChoiceAnswer.Incorrect);
        super.select();
    }
}
